data/json/morale_types.json

```diff
@@ -279,6 +279,11 @@
     "type": "morale_type",
     "text": "Filthy Gear"
   },
+  {
+    "id": "morale_butcher",
+    "type": "morale_type",
+    "text": "Memories of Butchering Human Corpse"
```

Comment:
Suggest `[Appalled/Anguished] for Butchering [Fellow] Human` --- [ ] are optional/options, "fellow" might not always be the case with cyborgs/supermutants/mycus.